client/src/index.jsx

```diff
@@ -149,6 +151,7 @@ class ApateezGallery extends React.Component {
       </div>
     );
   }
-} 
-  
-ReactDOM.render(< ApateezGallery/>, document.getElementById('app'));
+}
+
+// ReactDOM.render(< ApateezGallery/>, document.getElementById('app'));
+export default ApateezGallery;
```

Comment:
This may be personal preference but I feel like there's a lot of empty lines in this file that could be removed so the file is shorter